tests/correct/incrementAndDecrement.rock

```diff
@@ -9,10 +11,12 @@ say my world
 knock my world down down down down down down
```

Comment:
This could get a new section starting at line 8 with `(Additional ',up' and ',down' work as multiple increments and decrements)`